JitsiConference.js

```diff
@@ -1601,7 +1599,10 @@ JitsiConference.prototype.onCallEnded = function(
     if (jingleSession === this.jvbJingleSession) {
         this.wasStopped = true;
 
-        Statistics.sendEventToAll(SESSION_TERMINATE);
+        const event = createJingleEvent('terminate');
+
+        Statistics.sendLog(JSON.stringify(event));
```

Comment:
looks like we do these 2 things a lot, is it worth adding a helper to do both? `Statistics.logAndSendEvent` or something?